src/os.toit

```diff
@@ -26,3 +29,6 @@ env / EnvironmentVariableMap ::= EnvironmentVariableMap.private_
 
 get_env_ key/string -> string?:
   #primitive.core.get_env
+
+set_env_ key/string value/string? -> none:
```

Comment:
The `set_env_` seems to take `null` as value.
Does that mean that one can remove the value this way?
Add a `remove` method ?